lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2315,6 +2335,18 @@ function EDITOR:AC_Check( notimer )
 
 		local word, _ = self:AC_GetCurrentWord()
 
+		-- PickColorFind
```

Comment:
Just a Ctrl+F thing, I believe for this to work I have to use the avaliable functions, I'll remove those after this is finished.